Packs/Barikat-ClosedQradarAndXsoar/Scripts/BClosedQradarAndXsoar/BClosedQradarAndXsoar.yml

```diff
@@ -0,0 +1,22 @@
+args:
+- name: note
+  type: textArea
+commonfields:
+  id: B-ClosedQradarAndXsoar
+  version: -1
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ""
+dockerimage: demisto/python3:3.10.12.62631
+enabled: true
+name: B-ClosedQradarAndXsoar
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags: []
+type: python
+fromversion: 6.0.0
+tests:
```

Comment:
please add the `comment` field to the yml:
`comment: This script ........`

Reply:
comment: This script can be added as a button to the Cortex XSOAR layout. After conducting the review on the analyst layout screen, they can quickly enter an incident note and simultaneously close both QRadar offense and XSOAR investigation. With just one button, they can enter a note and perform the closing process in two places. Please note that this script is specifically designed for systems where QRadar and XSOAR work together.